Realm/Realm/Realm.cs

```diff
@@ -1323,10 +1322,12 @@ public void RemoveAll()
         /// non-null <see cref="RealmConfigurationBase.EncryptionKey"/>, the copy will be encrypted with that key.
         /// </summary>
         /// <remarks>
-        /// The destination file cannot already exist.
-        /// <para/>
-        /// If this is called from within a transaction it writes the current data, and not the data as it was when
-        /// the last transaction was committed.
+        /// 1. The destination file cannot already exist.
+        /// 2. For synchronized realms, the file written will have the client file identifier removed.
+        /// 3. Furthermore it is required that all local changes are synchronized with the server before the copy can be written.
+        ///    This is to be sure that the file can be used as a starting point for a newly installed application.
+        ///    The function will throw if there are pending uploads.
+        /// 4. It is advised to only copy a Realm file while it is not in use. Copying open Realms can lead to unexpected behvaiour.
```

Comment:
I don't think this comment is correct - copying an open file should be defined behavior, even if that behavior is to throw an error. Additionally, the way it's worded is quite confusing as `WriteCopy` is an instance method that can only be called on an open Realm. If the comment was taken verbatim, it would imply that users need to open a file, close it, then call `WriteCopy`, which will not work.

Reply:
6ec8b1f clarifies this difference and points out what happens for local vs. synced Realms.